src/components/quizOne.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<link rel="stylesheet" href="../styles/root.css" />
+	</head>
+	<body>
+		<div
+			class="InstallationDomMainContainer TypoHtag card-container-bg flex-space-evenly"
+		>
+			<div
+				class="padding-small-card shade classA translateOne"
+			>
+			 <div
+				class="padding-small-card shade flex-column-al-center translateOne margin-normal-bottom back-btn"
+		>score: 2/5 </div>
+					<div
+						class="quiz-container padding-small-card shade"
+					>
+						<p class="padding-normal">
+							Lorem ipsum, dolor sit amet consectetur
+							adipisicing elit. Eius cumque maiores a alias
+							repellat porro odit fugit possimus obcaecati
+							quasi quaerat adipisci iusto dignissimos, iure
+							quo reiciendis atque recusandae! Eveniet.
+						</p>
+					</div>
+					<div class="padding-normal card-selector-text">
+						<div class="option-container">
+							<button class="options quiz-container padding-small-card imgTransition" value="true">Option 1</button>
+							<button class="options quiz-container padding-small-card imgTransition">Option 2</button>
+							<button class="options quiz-container padding-small-card imgTransition">Option 3</button>
+							<button class="options quiz-container padding-small-card imgTransition">Option 4</li>
+						</div>
+					</div>
+
+					<a
+					class="col-12"
+					href="../components/Result.html"
+					target="iframe-quiz"
+				>
+					<div
+						class="padding-small-card shade flex-column-al-center translateOne margin-normal-top imgTransition back-btn"
+					>
+						Result
+					</div>
+				</a>
+
+					<div
+					class="padding-small-card shade flex-column-al-center translateOne margin-normal-top back-btn imgTransition col-12"
+					onclick="history.back()"
```

Comment:
is the onclick required here? gaurav once mentioned to avoid using events in HTML, you can ask him about it as I don't remember the reason. I am attaching a link as well: https://stackoverflow.com/questions/6941483/onclick-vs-event-handler

Reply:
There is back function provided by JS that stores your history from the browser and helps you to go back to visited pages. I'll do that here because I don't want to write a JS function for one line of code. 

I'll surely look into this Thank you Partha for the detailed PR review.